sdk/src/main/java/io/dapr/utils/NetworkUtils.java

```diff
@@ -47,4 +49,12 @@ public static void waitForSocket(String host, int port, int timeoutInMillisecond
       }
     }, timeoutInMilliseconds);
   }
+
+  /**
+   * Retrieve loopback address for the host.
+   * @return The loopback address String
+   */
+  public static String getHostLoopbackAddress() {
+    return InetAddress.getLoopbackAddress().getHostAddress();
```

Comment:
Can this method automatically return the Ipv6 between "[]"?

Reply:
@artursouza  Sorry for the delay, finally had some time to look into this. I changed some unit tests locally to validate IPv6 sidecar IP constants contained within "[]" and it seems like your suggestion is viable - for both URL endpoint invocation formats and Dapr client instantiation. Great recommendation!

However, before I proceed, I need to validate an implementation detail with you. To test for whether an address is IPv6, it would be convenient to leverage the Apache Commons Validator, though you have recommended against bringing in 3P libraries beyond the test scope. Is it possible to get an exception for this case, or do I have to write a custom IPv6 validator within NetworkUtils?

Reply:
@willtsai Is it complex to write a validator for IPv6 ? both standard pattern and hex compressed pattern along with IPv4 validation? 
I am wary of adding a new dependency specifically for one validation ....

Reply:
Understood - should not be too complex but I wanted to see if the work (and subsequent unit tests) could be avoided. Seems like adding a new dependency is not worth it here, so I'll work on the custom validator.